app/models/tank.rb

```diff
@@ -2,11 +2,11 @@ class Tank < ApplicationRecord
   belongs_to :user
   has_many :fish
 
-  def self.new_tank
+  def new_tank
     Tank.new
   end
 
-  def self.new_fish
+  def new_fish
```

Comment:
makes much more sense this way! its cool we did it a very "hard coded" style... but this is easier on me noggin